.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 npm-debug.log
 coverage
 .DS_Store
+package-lock.json
```

Comment:
Npm definitely wants this file to be checked in:
```
> npm install
npm notice created a lockfile as package-lock.json. You should commit this file.
```
Would it be a big deal if the package-lock.json and yarn.lock were not precisely the same (given they were semver compatible)?